packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2430,7 +2434,7 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
     final _Decorator decorator = _Decorator(
       decoration: _Decoration(
         contentPadding: contentPadding,
-        isCollapsed: decoration.isCollapsed,
+        isCollapsed: decoration.isCollapsed ?? themeData.inputDecorationTheme.isCollapsed,
```

Comment:
Nit: Would it make sense to create a getter for this somewhere?  I say that in order to try to make it more likely that future people working on this file will remember to use both values of isCollapsed.

Totally up to you though, I'm not sure it's actually better with a getter.

Reply:
You could get value of `isCollapsed` by using `InputDecorator().isCollapsed` or `Theme.of(context).inputDecorationTheme.isCollapsed`.  
I don't really think it'd be really necessary for an explicit getter right now...